src/spikeinterface/widgets/unit_summary.py

```diff
@@ -70,6 +81,13 @@ def plot_matplotlib(self, data_plot, **backend_kwargs):
         unit_colors = dp.unit_colors
         sparsity = dp.sparsity
 
+        widget_params = defaultdict(lambda: dict(), dp.widget_params)
```

Comment:
cool! I did not know about that `defaultdict`. Might be adding a quick comment like `# Returns an empty dict if the key is not in passed in dp.widget_params`, but maybe this is quite widely known and doesn't need the comment

Reply:
Just added a comment :)